### PR TITLE
Add street type

### PR DESCRIPTION
**Pull request by mattjdev**

### CODE REVIEW

**Comment by alexkwolfe** [comment]
LGTM. What do you think @cgrayson?

File: src/street.coffee [hunk 1]
```diff
@@ -0,0 +1,51 @@
+_ = require('lodash')
+phoneUtil = require('libphonenumber').phoneUtil
```
Comment (cgrayson) [inline src/street.coffee]: This isn't used here, is it?

Comment (mattjdev) [inline src/street.coffee]: oops. nope. Now you see where I copied the code from... and now you don't! :magic_wand:

File: src/street.coffee [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+  parsed
+
+
+module.exports =
+  parse: parse
+  components: [
+    { name: 'raw', type: 'string', description: 'Unmodified value' }
+    { name: 'street_number', type: 'string', description: 'Street Number' }
+    { name: 'street_name', type: 'string', description: 'Street Name' }
```
Comment (cgrayson) [inline src/street.coffee]: Would shorter component names of `number` and `name` be just as clear?

**Comment by cgrayson** [comment]
Otherwise LGTM2

**Pull request merged**